md_rahaman/python/django/random_word_generator/apps/random_word_generator_app/templates/random_word_generator_app/index.html

```diff
@@ -0,0 +1,22 @@
+
+<html>
+	<head>
+		<title>Random World Generator</title>
+		<style type="text/css">
```

Comment:
Internal css... this is a no no. Don't get too lazy with your code development. Practice makes perfect.